R/utils_memoised.R

```diff
@@ -35,7 +29,7 @@ memoise_cran_mirrors <- memoise::memoise({
   # NOTE: might need to implement actual caching to avoid inconsistent behavior
   # when run with spotty internet
   function(all = TRUE, ...,
-    host_exists = !is.null(curl::nslookup("cran.r-project.org", error = FALSE))) {
+           host_exists = !is.null(curl::nslookup("cran.r-project.org", error = FALSE))) {
```

Comment:
Not my preferred style, but until (if?) we impose a style with lintr I think we can accept anything sensible. 😛

Reply:
agree, we can have an issue open to go through lintr for a global check.